pcsx2/DEV9/ATA/ATA_Info.cpp

```diff
@@ -0,0 +1,391 @@
+/*  PCSX2 - PS2 Emulator for PCs
+ *  Copyright (C) 2002-2020  PCSX2 Dev Team
+ *
+ *  PCSX2 is free software: you can redistribute it and/or modify it under the terms
+ *  of the GNU Lesser General Public License as published by the Free Software Found-
+ *  ation, either version 3 of the License, or (at your option) any later version.
+ *
+ *  PCSX2 is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+ *  PURPOSE.  See the GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along with PCSX2.
+ *  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "PrecompiledHeader.h"
+
+#include "ATA.h"
+#include "../DEV9.h"
+
+void WriteUInt16(u8* data, int* index, u16 value)
+{
+	*(u16*)&data[*index] = value;
+	*index += sizeof(value);
+}
+void WriteUInt32(u8* data, int* index, u32 value)
+{
+	*(u32*)&data[*index] = value;
+	*index += sizeof(value);
+}
+void WriteUInt64(u8* data, int* index, u64 value)
```

Comment:
Why not encapsulate them into a DEV9 namespace?

Reply:
As in a `namespace DEV9` block? or something else? 

Didn't think anything in DEV9 was in a namespace

Reply:
Ah right, they're not because I didn't get around to redesign the plugin API yet. Can you move them in the ATA namespace, at least temporarily?

Reply:
I'll stick it into the class, as there isn't a ATA namespace either